tests/users/test_api_resources.py

```diff
@@ -343,6 +344,39 @@ def test_user_registration_with_taken_email(self, send_email_function):
             )
             self.assertEqual(response.status_code, 409)
 
+    @patch("flask_mail._MailMixin.send", side_effect=mail_send_mocked)
+    def test_user_registration_race(self, send_email_function):
+        user = UserModel.query.first()
+        email = user.email
+        results = []
+
+        def request(x):
```

Comment:
@epicadk where you use this argument `x` passed in request

Reply:
It was used for debugging I'll remove it.